app/controllers/item_orders_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ItemOrdersController <ApplicationController
+
+  def show
+    @item_order = ItemOrder.find(item_order_params[:item_order_id])
+  end
+
+  def update
+    # require "pry"; binding.pry
```

Comment:
Ugh. My bad for not catching that commented pry before pushing.

Reply:
C'mon man

Reply:
Ha. That's fine. We can do a clean-up branch later